hawkeye-api/src/handlers.rs

```diff
@@ -449,41 +466,45 @@ pub async fn stop_watcher(id: String, client: Client) -> Result<impl warp::Reply
         )),
         Status::Pending => Ok(reply::with_status(
             reply::json(&json!({
-                "message": "Watcher is updating"
+                "message": "Watcher is already updating"
             })),
             StatusCode::CONFLICT,
         )),
         Status::Running => {
             // Stop watcher / replicas to 0
-            let patch_params = PatchParams::default();
+            let mut patch_params = PatchParams::default();
+            patch_params.field_manager = Option::from("hawkeye_api".to_string());
```

Comment:
Same as before, I think could be better to
```suggestion
            patch_params.field_manager = Some("hawkeye_api".to_string());
```